packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -19,7 +25,7 @@ export interface IButtonState {
 
 export class Button extends BaseComponent<IButtonProps, IButtonState> implements IButton {
```

Comment:
Are subclasses the right way to go, or should `PrimaryButton` etc. be the result of the invocation of a hoc? 

Something like making `rootClassName` prop on `ButtonBase` and then:
`export const PrimaryButton = withRootClassName(ButtonBase, "primary-button")` ? Just a thought.